Ducky-Chickens/bangazonCLI
# Select Active Customer

# Description
1. Adds option number 2 from the main menu, activate a customer.  
1. Removed the out folder form the project and git ignored it.  We can generate that any time we want to with the jsdoc command.

## Related Ticket(s)
Completes Issue #2 (User should be able to select who the current customer is)

## Expected Behavior
On the main menu, when you select option 2, you then can select a customer by there id.  You cannot select a customer that does not exist.

## Steps to Test Solution
1. Run `db:reset`
1. Run `npm i -g`
1. Run `bangazon`
1. Select option 2 `2`
1. Enter any id that is listed.  If successful, the program should end.
     1. Attempt numbers that are not on the list.  The prompt should ask you for the id again if the id does not exist on the list.

1. Run `npm test`
    1. Verify that all test are checked.  They should be.

## Testing
